src/views/evaluation/Evaluation.js

```diff
@@ -0,0 +1,122 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
+    return `
+    <li class="header-elements ${specialPlacement}">
+        <a class = "header-elements-link header-elements-linkT ${active} " href="${option.Link}"> ${option.Text}</a>
+    </li>
+    `
+}
+
+
+const constructNav = function(option = {}){
+    const currentPage = "Evaluationv2.html", result =[], specialPlacement = "Loginv2.html";
+    for(var i=0;i<option.Link.length;i++){
+        if(option.Link[i] === currentPage){
+            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+        }else{ 
+            if(option.Link[i] === specialPlacement){
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "", "right-button"))
```

Comment:
I'd argue that this piece of code is very similar to L29 and L34, except for very small differences. This brings me to one of the most helpful principles in software development, namely the DRY - Don't Repeat Yourself - principle. If you find yourself writing the same code over and over again or, even worse, using the eternally damning combo of <kbd>Ctrl + C</kbd> / <kbd>Ctrl + V</kbd>, stop and think about how you can abstract your code in such a way that you only have to modify it in a single place (as opposed to throughout the code base, potentially - though hopefully not -
 in dozens of files), should you ever need to do so.